internal/appsec/remoteconfig.go

```diff
@@ -409,7 +410,14 @@ func (a *appsec) enableRASP() {
 	if err := remoteconfig.RegisterCapability(remoteconfig.ASMRASPSSRF); err != nil {
 		log.Debug("appsec: Remote config: couldn't register RASP SSRF: %v", err)
 	}
-	// TODO: register other RASP capabilities when supported
+	if err := remoteconfig.RegisterCapability(remoteconfig.ASMRASPSQLI); err != nil {
```

Comment:
WDYT of moving the calls to `RegisterCapability` into the WAF listener's instantiations, like setting the SQLi capability when the SQL listener is being setup?